crypto/cipher/aes_gcm_ossl.c

```diff
@@ -207,35 +208,40 @@ static srtp_err_status_t srtp_aes_gcm_openssl_context_init (void* cv, const uint
     return (srtp_err_status_ok);
 }
 
-
 /*
  * aes_gcm_openssl_set_iv(c, iv) sets the counter value to the exor of iv with
  * the offset
  */
-static srtp_err_status_t srtp_aes_gcm_openssl_set_iv (void *cv, uint8_t *iv, srtp_cipher_direction_t direction)
+static srtp_err_status_t srtp_aes_gcm_openssl_set_iv(
+    void *cv,
+    uint8_t *iv,
+    srtp_cipher_direction_t direction)
```

Comment:
When parameters are to long to align with end of function they will be moved to start of line. Talked to @thisisG  about this and we agree it is an OK exception for now.